src/store/utils.ts

```diff
@@ -42,7 +42,14 @@ export function createApiRequest<
         });
 
         try {
-            const result = await request;
+            let result;
+
+            if (Array.isArray(request)) {
+                result = await Promise.all(request);
```

Comment:
In case any of the requests fails, the catch statement would receive only the first error. Potentially, this could lead to unevident bugs. E.g. out of two requests the first one fails because of a transport error, whereas the second one answers with HTTP 401. This should result in the auth form displayed, but will only produce an error toast.